### PR TITLE
Add documentation for evaluation script update

### PR DESCRIPTION
**Pull request by Ram81**
### Description

Add documentation for updating the evaluation script on read the docs

### CODE REVIEW

**Comment by Ram81** [comment]
@RishabhJain2018 how do we test this before deploying? do we have a staging setup for rtd?

File: docs/source/edit_evaluation_script.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## Editing Evaluation Script
+
+Each challenge has an evaluation script, which evaluates the submission of participants and returns the scores which will populate the leaderboard. The logic for evaluating and judging a submission is customizable and varies from challenge to challenge.
```
Comment (RishabhJain2018) [inline docs/source/edit_evaluation_script.md:3]: `Each challenge` -> `Each prediction upload challenge`

Comment (RishabhJain2018) [inline docs/source/edit_evaluation_script.md:3]: `and judging`, please remove this.

File: docs/source/edit_evaluation_script.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+When setting up a new challenge hosts need to test multiple versions of evaluation script on EvalAI. To test multiple versions of evaluation script host can update the evaluation script of existing challenge without uploading a whole new challenge configuration.
```
Comment (RishabhJain2018) [inline docs/source/edit_evaluation_script.md:5]: `When setting up a new challenge` -> `When setting up a new challenge,`

File: docs/source/edit_evaluation_script.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+<img src="_static/img/evaluation_criteria_tab.png"/>
+
+### 3. Update the evaluation script
+
+Upload the latest evaluation script and click on 'Submit' button to update the evaluation script
+
+<img src="_static/img/edit_evaluation_script.png"/>
+
+**Tada!** you have successfully updated the evaluation script for a challenge. The evaluation workers for the challenge will be restarted automatically to pick up the latest evaluation script.
```
Comment (RishabhJain2018) [inline docs/source/edit_evaluation_script.md:27]: `The evaluation workers for the challenge will be restarted automatically to pick up the latest evaluation script.` -> `The evaluation workers for the challenge will be restarted automatically to pick up the latest evaluation script. Please wait for a minimum of 10 minutes for the workers to restart.`

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]
Can we please use red bounding boxes in all the images?

**Comment by RishabhJain2018** [comment]
@Ram81 Can you please update this PR.

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**